test_pvc_snapshot[CephFileSystem] failed waiting for PVC from snapshot

The test case tests/functional/pv/pvc_snapshot/test_pvc_snapshot.py::TestPvcSnapshot::test_pvc_snapshot[CephFileSystem] failed because the creation of PVC from snapshot took more time than the given timeout of 60 seconds.

Logs - https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/632/18161/888701/888727/log

```
restore_pvc_obj = pvc.create_restore_pvc(
        sc_name=sc_name,
        snap_name=snap_obj.name,
        namespace=snap_obj.namespace,
        size=pvc_size,
        pvc_name=restore_pvc_name,
        restore_pvc_yaml=restore_pvc_yaml,
    )

  helpers.wait_for_resource_state(restore_pvc_obj, constants.STATUS_BOUND)
```

```

E           ocs_ci.ocs.exceptions.ResourceWrongStatusException: Resource restore-pvc-test-e48049f145fe4553910035d describe output: Name:          restore-pvc-test-e48049f145fe4553910035d
E           Namespace:     namespace-test-a5c7622c651b4521bdf1db44a
E           StorageClass:  ocs-storagecluster-cephfs
E           Status:        Pending
E           Volume:
E           Labels:        
E           Annotations:   volume.beta.kubernetes.io/storage-provisioner: openshift-storage.cephfs.csi.ceph.com
E                          volume.kubernetes.io/storage-provisioner: openshift-storage.cephfs.csi.ceph.com
E           Finalizers:    [kubernetes.io/pvc-protection]
E           Capacity:
E           Access Modes:
E           VolumeMode:    Filesystem
E           DataSource:
E             APIGroup:  snapshot.storage.k8s.io
E             Kind:      VolumeSnapshot
E             Name:      snapshot-test-7b0fb5f37f82458a9e6341d981
E           Used By:     
E           Events:
E             Type     Reason                Age                From                                                                                                                      Message
E             ----     ------                ----               ----                                                                                                                      -------
E             Warning  ProvisioningFailed    66s                openshift-storage.cephfs.csi.ceph.com_csi-cephfsplugin-provisioner-6bdc4c7cc4-7vtbf_80a0f4c4-aa0a-4078-a5ee-389653781828  failed to provision volume with StorageClass "ocs-storagecluster-cephfs": rpc error: code = Aborted desc = clone from snapshot is pending
E             Warning  ProvisioningFailed    33s (x6 over 64s)  openshift-storage.cephfs.csi.ceph.com_csi-cephfsplugin-provisioner-6bdc4c7cc4-7vtbf_80a0f4c4-aa0a-4078-a5ee-389653781828  failed to provision volume with StorageClass "ocs-storagecluster-cephfs": rpc error: code = Aborted desc = clone from snapshot is already in progress
E             Normal   ExternalProvisioning  2s (x7 over 66s)   persistentvolume-controller                                                                                               Waiting for a volume to be created either by the external provisioner 'openshift-storage.cephfs.csi.ceph.com' or manually by the system administrator. If volume creation is delayed, please verify that the provisioner is running and correctly registered.
E             Normal   Provisioning          1s (x8 over 66s)   openshift-storage.cephfs.csi.ceph.com_csi-cephfsplugin-provisioner-6bdc4c7cc4-7vtbf_80a0f4c4-aa0a-4078-a5ee-389653781828  External provisioner is provisioning volume for claim "namespace-test-a5c7622c651b4521bdf1db44a/restore-pvc-test-e48049f145fe4553910035d"



ocs_ci/helpers/helpers.py:123: ResourceWrongStatusException
```

--- a/tests/functional/pv/pvc_snapshot/test_pvc_snapshot.py
+++ b/tests/functional/pv/pvc_snapshot/test_pvc_snapshot.py
@@ -136,7 +136,9 @@ class TestPvcSnapshot(ManageTest):
             pvc_name=restore_pvc_name,
             restore_pvc_yaml=restore_pvc_yaml,
         )
-        helpers.wait_for_resource_state(restore_pvc_obj, constants.STATUS_BOUND)
+        helpers.wait_for_resource_state(
+            restore_pvc_obj, constants.STATUS_BOUND, timeout=90
+        )
         restore_pvc_obj.reload()
         teardown_factory(restore_pvc_obj)
 

